src/index.ts

```diff
@@ -209,12 +209,10 @@ function updateAllPos() {
  * random offset.
  * @param el - Element
  */
+let intervalIds: NodeJS.Timeout[] = [];
 function poll(el: Element) {
   setTimeout(() => {
-    intervals.set(
-      el,
-      setInterval(() => lowPriority(updatePos.bind(null, el)), 2000)
-    )
+    intervalIds.push(setInterval(() => lowPriority(updatePos.bind(null, el)), 2000));
```

Comment:
These intervals are not scoped to the current element. This means if any unmount is called all the intervals for any animations on the page will be cleared. We should continue to use the intervals WeakSet.